build.sbt

```diff
@@ -50,6 +50,7 @@ lazy val root =
     logBuffered := false,
     parallelExecution in Benchmark := false
 )
+publishArtifact in Benchmark := false
```

Comment:
Accidentally forgot to push this part of the change previously. This line ensures the scalameter dependency gets marked as `test` scope in the published pom.

Reply:
Oh, sorry, I haven't noticed that.